src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicCipherSuitesPolicyTests.cs

```diff
@@ -71,10 +71,12 @@ public void NoSupportedCiphers_ThrowsArgumentException(TlsCipherSuite[] ciphers)
         [Fact]
         public async Task MismatchedCipherPolicies_ConnectAsync_ThrowsQuicException()
         {
-            await Assert.ThrowsAsync<QuicException>(() => TestConnection(
+            QuicException ex = await Assert.ThrowsAsync<QuicException>(() => TestConnection(
```

Comment:
`There is a helper for this, something like `AssertThrowsQuicException(QuicError.TransportError, () => ...);`

Reply:
updated.